liquidctl/cli.py

```diff
@@ -201,6 +201,8 @@ def _print_dev_status(dev, status):
         if isinstance(v, datetime.timedelta):
             v = str(v)
             u = ''
+        elif v is None:
+            v = 'N/A'
```

Comment:
I wasn't sure if I should set `u = ''` or not (in the general case).

Reply:
Do you mean also set the units to `''` when the value is `None`?

Reply:
Exactly, in case we ever get something like a temperature that's N/A.

In this hypothetical scenario `get_status()` would probably return `("That temperature", None, "°C")`, as that's the natural thing to do inside a driver.  But what should we output for `--json`?

Reply:
That situation happens for the commander pro if the status is requested before the device is initialized, well currently it returns 0 for everything.  (it was an implementation decision that I decided to make, although that could have been a bad one to make).

for JSON we could make the value `null` which is valid in the spec or coerce it to a `0`? although I don't like the 0 idea as much

Reply:
Sorry, I don't know why I mentioned `--json`, it has nothing to do with this PR.

Instead, what should we output here (for humans, not `--json`)?

```
That temperature        N/A  °C
```

or

```
That temperature        N/A  
```

For consistency with other (presumably available) values, I prefer the first one, but I'm not 100% sure.

Reply:
Re. the Commander Pro, if we're 100% sure that the device hasn't been initialized, then a warning + `None` (N/A for humans) could be an improvement, yes.  But only if we're certain that's not what the device is actually reporting... if the device is initialized and reports 0 than we should keep that zero.

Reply:
> Instead, what should we output here ...

I kind of like the second one, but the first is more consistent so I think that is probably a better choice to go with